### DISCUSSION TITLE
Loading Issue on http://127.0.0.1:5000

### DISCUSSION THREAD

**Discussion by ahnafaf** [veeqtoh/linkedin-jobs-scraper#4] [General]
I think there might be a bug, as the webpage loads forever. Is this supposed to be the use case? The scrapping works.

Tried solutions: None

**Comment by veeqtoh** [comment]
Yes, This is expected with the current version. 

As the page loads on visiting http://127.0.0.1:5000, the scrapper is activated. If you checked `data/linkedin-jobs.csv` directory, you should see the jobs populated.

In coming versions, I intend to make the landing page more interactive.
Let me know if there's anything else I can do to help.

**Comment by ahnafaf** [comment]
Thank you so much, I appreciate the help.
